client/recordUtil.js

```diff
@@ -262,6 +273,12 @@ const mergeRecords = (recordsAndObjects) => {
  */
 module.exports.resolveRecords = (recordsAndExistingObjects) => {
   let resolvedRecords = []
+  recordsAndExistingObjects.forEach((item) => {
+    if (item) {
```

Comment:
maybe not necessary at the moment but better to be safe here since resolveRecords is an exported function